.github/CODEOWNERS

```diff
@@ -1,2 +1 @@
-# This should match the team set up in https://github.com/orgs/opensearch-project/teams and include any additional contributors
-*   @opensearch-project/clients @robsears
\ No newline at end of file
+* @VijayanB @jayeshathila @robsears @vamshin @yuvi17 @dblock @VachaShah @nhtruong
```

Comment:
@yuvi17 is no longer working on this project and doesn't have access AFAIK, should they be moved to emeritus and removed from CODEOWNERS?